src/client/scenes/universeloadingscene.cpp

```diff
@@ -104,7 +109,7 @@ void cqsp::scene::UniverseLoadingScene::LoadUniverse() {
     script_interface.RegisterDataGroup("events");
 
     using cqsp::asset::TextAsset;
-    // Process scripts for core
+    // Process scripts for cores(loading the scripts?)
```

Comment:
Remove all the changes in this file, you didn't really change anything code wise, so just revert his file so that we don't have so many files to review.

Reply:
I think that comment was really unclear sorry

Reply:
Can you revert all the changes in this file, none of the changes here were necessary

Reply:
sorry, but that comment was too ambigous